modules/mono/glue/cs_files/Mathf.cs

```diff
@@ -208,6 +321,21 @@ public static real_t Rad2Deg(real_t rad)
             return rad * Rad2DegConst;
         }
 
+        public static real_t Rem(real_t a, real_t b) 
+        {
+            return (a % b);
+        }
+
+        public static int RemInt(real_t a, real_t b) 
+        {
+            return RoundToInt(a % b);
+        }
+
+        public static int RemInt(int a, int b) 
+        {
+            return (a % b);
```

Comment:
I dont understand the usefullness of this function, by this logic you must also implement add(int a, int b), subract(int a, int b) etc... its redundant, because its basic operation, and its not a problem for user to write "a % b" himself

Reply:
It is useful because `%` is ambiguous and has different meanings in different programming languages. In Python, % is Mod, but in C-derived languages, it's Rem. But as they are equivalent for positive values, programmers may not be aware of the different behavior with negative numbers. `Mathf.Rem()` allows saying "I explicitly want Remainder and not Modulus".